server/odc-service/src/main/java/com/oceanbase/odc/service/common/DruidMonitor.java

```diff
@@ -0,0 +1,646 @@
+        }
+
+        public String buildTooLongAvgExecuteTimeMsg() {
+            return getMethodIdentification() + ", avgExecuteTime=" + computeAvgExecuteTime() + ";";
+        }
+
+        public String buildTooLongJdbcAvgExecuteTimeMsg() {
+            return getMethodIdentification() + ", avgJdbcExecuteTime=" + computeAvgJdbcExecuteTime() + ";";
+        }
+
+        public String buildTooMuchJdbcExecuteCountMsg() {
+            return getMethodIdentification() + ", avgJdbcExecuteCount=" + computeAvgJdbcExecuteCount()
+                    + ";";
+        }
+
+        public static DruidMethodStatVO mapToVO(Map<String, Object> map) {
+            DruidMethodStatVO vo = new DruidMethodStatVO();
+            vo.setClazz((String) map.get(DruidMethodStatConstants.CLASS));
+            vo.setMethod((String) map.get(DruidMethodStatConstants.METHOD));
+            vo.setRunningCount((Integer) map.get(DruidMethodStatConstants.RUNNING_COUNT));
+            vo.setConcurrentMax((Integer) map.get(DruidMethodStatConstants.CONCURRENT_MAX));
+            vo.setExecuteCount((Long) map.get(DruidMethodStatConstants.EXECUTE_COUNT));
+            vo.setExecuteErrorCount((Long) map.get(DruidMethodStatConstants.EXECUTE_ERROR_COUNT));
+            vo.setExecuteTimeMillis((Long) map.get(DruidMethodStatConstants.EXECUTE_TIME_MILLIS));
+            vo.setJdbcCommitCount((Long) map.get(DruidMethodStatConstants.JDBC_COMMIT_COUNT));
+            vo.setJdbcReadOnlyCount((Long) map.get(DruidMethodStatConstants.JDBC_READ_ONLY_COUNT));
+            vo.setJdbcRollbackCount((Long) map.get(DruidMethodStatConstants.JDBC_ROLLBACK_COUNT));
+            vo.setJdbcPoolConnectionOpenCount((Long) map.get(DruidMethodStatConstants.JDBC_POOL_CONNECTION_OPEN_COUNT));
+            vo.setJdbcPoolConnectionCloseCount(
+                    (Long) map.get(DruidMethodStatConstants.JDBC_POOL_CONNECTION_CLOSE_COUNT));
+            vo.setJdbcResultSetOpenCount((Long) map.get(DruidMethodStatConstants.JDBC_RESULT_SET_OPEN_COUNT));
+            vo.setJdbcResultSetCloseCount((Long) map.get(DruidMethodStatConstants.JDBC_RESULT_SET_CLOSE_COUNT));
+            vo.setJdbcExecuteCount((Long) map.get(DruidMethodStatConstants.JDBC_EXECUTE_COUNT));
+            vo.setJdbcExecuteErrorCount((Long) map.get(DruidMethodStatConstants.JDBC_EXECUTE_ERROR_COUNT));
+            vo.setJdbcExecuteTimeMillis((Long) map.get(DruidMethodStatConstants.JDBC_EXECUTE_TIME_MILLIS));
+            vo.setJdbcFetchRowCount((Long) map.get(DruidMethodStatConstants.JDBC_FETCH_ROW_COUNT));
+            vo.setJdbcUpdateCount((Long) map.get(DruidMethodStatConstants.JDBC_UPDATE_COUNT));
+            vo.setLastError(map.get(DruidMethodStatConstants.LAST_ERROR));
+            vo.setLastErrorTime((Date) map.get(DruidMethodStatConstants.LAST_ERROR_TIME));
+            vo.setHistogram((long[]) map.get(DruidMethodStatConstants.HISTOGRAM));
+            return vo;
+        }
+    }
+
+    public static String compressSql(String sql) {
```

Comment:
we may add a method to do this, how about 'StringUtils#getBriefSql'?

Reply:
done